doc/manual/source/man/dnst-signzone.rst

```diff
@@ -0,0 +1,108 @@
+dnst-signzone
+===============
+
+Synopsis
+--------
+
+:program:`dnst signzone` [``OPTIONS``] ``<ZONEFILE>`` ``<KEY>...``
+
+Description
+-----------
+
+**dnst signzone** signs the zone with the given key(s).
```

Comment:
For consistency and accuracy I think this should say `zonefile` like we use below.

Reply:
I guess technically it signs the _zone in the zonefile_?